astropy/io/fits/tests/test_connect.py

```diff
@@ -92,13 +92,14 @@ def test_simple_noextension(self, tmpdir):
         t2 = Table.read(filename)
         assert equal_data(t1, t2)
 
-    def test_with_units(self, tmpdir):
+    @pytest.mark.parametrize('table_type', (Table, QTable))
```

Comment:
Yes, honestly a silly mistake. Had done the changes without local tests. Have made the changes and tested them as well.